tasks/collect-data/tests/test-collect-data-fail-missing-cr.yaml

```diff
@@ -7,15 +7,18 @@ metadata:
     test/assert-task-failure: "run-task"
 spec:
   description: |
-    Run the collect-data task without a snapshot cr and verify that the task fails as expected.
+    Run the collect-data task without a ReleasePlanAdmission cr and verify that the task fails as expected.
+    This test cannot rely on the snapshot not existing as the snapshot is retrieved with a jsonpath added
```

Comment:
@davidmogar @mmalina this relates to Martin's comment on the utils PR. Is this new behavior okay with us?